extension/inet/inet_functions.cpp

```diff
@@ -3,13 +3,65 @@
 #include "duckdb/common/string_util.hpp"
 #include "duckdb/common/pair.hpp"
 #include "duckdb/common/operator/cast_operators.hpp"
+#include "duckdb/common/operator/subtract.hpp"
+#include "duckdb/common/operator/add.hpp"
 #include "duckdb/common/types/cast_helpers.hpp"
 #include "duckdb/common/vector_operations/generic_executor.hpp"
 
+/* warning suppress begin */
+/* Only a few functions out of numeric_cast are used*/
+#if defined(__clang__)
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunused-function"
+#elif defined(__GNUC__)
+#if (__GNUC__ > 4) || (__GNUC__ == 4 && __GNUC_MINOR__ >= 6)
+#pragma GCC diagnostic push
+#endif
+#pragma GCC diagnostic ignored "-Wunused-function"
+#endif
+
+#include "duckdb/common/operator/numeric_cast.hpp"
```

Comment:
Is this include here necessary at all? Can we perhaps include `duckdb/common/operator/cast_operators.hpp` instead? Then we might also be able to get rid of all of the pragmas.

Reply:
Turns out it isn't necessary, uhugeint has a built in cast operator from hugeint, and going from always positive hugeint to uhugeint has no risk of overflow, so the lack of checks isn't a concern.